lib/html-proofer/element.rb

```diff
@@ -77,11 +79,11 @@ def path
     end
 
     def hash
-      parts.fragment unless parts.nil?
+      parts&.fragment
```

Comment:
Wouldn't safe navigation fail for ruby < 2.3.0? and if so, is a minimum version of ruby specified anywhere? cc:	@gjtorikian

Reply:
< 2.3 is EOL (in fact, 2.4 is EOL soon) and I don't support dead rubies.